cloud/misc/proxmox_kvm.py

```diff
@@ -604,6 +604,19 @@
     name        : spynal
     node        : sabrewulf
     state       : current
+
+# Clone VM from template ( or another VM )
+- proxmox_kvm:
+       api_user    : tfm@pve
+       api_password: secret
+       api_host    : helldorado
+       clone       : 555
+       vmid        : 666
+       node        : proxmox2
+       name        : onetest
+       storage     : 'stor2'
+       format      : 'qcow2'
+       timeout     : 300
```

Comment:
I used timeout 300 because the default timeout was 30 and in my case the full vm clone was not done in 30 seconds ( took about 70 seconds for my 16 gb vm but i have a slower dev environment).